proteomics.py

```diff
@@ -5,18 +5,16 @@
 import re
 import binascii
 
-from galaxy.datatypes.sniff import *
```

Comment:
Not 100% sure about this. Importing `*` is is considered bad practise because you have hard times to track back what is imported and used. I haven't found any function that is used. So hoping this is fine.